azure-client-runtime/src/test/java/com/microsoft/azure/v2/PagedListTests.java

```diff
@@ -279,30 +279,30 @@ public void canCreateObservableFromPagedList() {
                 return firstObservable().concatWith(nextObservable());
             }
 
-            rx.Observable<Integer> firstObservable() {
-                return rx.Observable.defer(new Func0<rx.Observable<Integer>>() {
+            io.reactivex.Observable<Integer> firstObservable() {
```

Comment:
Why do we need to include the package here?

Reply:
We don't. Seems like it ended up there by mistake. (is there checkstyle for qualified access when the class has been imported already?)